src/gui/plugins/scene3d/GzScene3D.qml

```diff
@@ -98,4 +99,29 @@ Rectangle {
       GzScene3D.OnDropped(drop.text, drag.x, drag.y)
     }
   }
+
+  // pop error
+  Connections {
+    target: GzScene3D
+    onPopupError: errorPopup.open()
+  }
+
+  Dialog {
+    id: errorPopup
+    parent: ApplicationWindow.overlay
+    modal: true
+    focus: true
+    x: (parent.width - width) / 2
+    y: (parent.height - height) / 2
+    background: Rectangle {
+      border.width: 5
+      border.color: "red"
```

Comment:
Not sure if you meant for the red border to be constant around the dialog. The thickness changes halfway:

![image](https://user-images.githubusercontent.com/5751272/128439158-6fb2e35d-5264-4bce-9882-a6928e5c2b62.png)

I personally think that the border is a bit too much, I don't remember seeing it on errors using other apps. The material design guide shows a plain dialog for errors: https://material.io/archive/guidelines/patterns/errors.html#errors-app-errors

Reply:
Removed 0fd716f68afe0a4691e42b43956b40880e9de545